include/zephyr/net/net_ip.h

```diff
@@ -1709,6 +1709,47 @@ static inline uint8_t net_priority2vlan(enum net_priority priority)
  */
 const char *net_family2str(sa_family_t family);
 
+#if defined(CONFIG_NET_IPV6_PE)
```

Comment:
Do we need to do something special for `doxygen`?

Reply:
Good point, yes we do.